Initial PR for repo (#1)
Vualto/offline-shaka: README.md

# Offline Shaka Player - DRM Media Playback Guide

This guide will show you how to setup Shaka Player and test DRM encrypted media offline.

On line 2 in offlineVualto.js , replace the const token value to a token generated by the VUDRM admin site. Ensure that {“liccache”:”yes”} is added to the Policy before generating the token. This will enable the Widevine license to be used with offline content.

To test offline DRM encrypted playback, https is needed due to Chrome using EME. The easiest way to set this up is to follow the below:

- Ensure Visual Studio Code is installed.

- Install the “Live Server” extension: https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer

> To enable https, you must create a certificate to self authenticate for local host: 

Reviewer: ```suggestion
To enable https, you must create a certificate to self authenticate for localhost: 
```

Author: Actioned.

> - Create a private key - ensure openssl is installed and input the following command: openssl genrsa -aes256 -out localhost.key 2048 

Reviewer: ```suggestion
- Create a private key - ensure openssl is installed and input the following command: `openssl genrsa -aes256 -out localhost.key 2048`
```

Author: Actioned.

- Provide a password. This will create the localhost.key.

> - Create the certificate by inputting the following command: openssl req -days 3650 -new -newkey rsa:2048 -key localhost.key -x509 -out localhost.pem

Reviewer: ```suggestion
- Create the certificate by inputting the following command: `openssl req -days 3650 -new -newkey rsa:2048 -key localhost.key -x509 -out localhost.pem`
```

Author: Actioned.

> - Input the password from the above, complete all prompts.This creates locahost.pem 

Reviewer: ```suggestion
- Input the password from the above, complete all prompts. This creates locahost.pem.
```

Author: Actioned.

- In the root of the project, create a .vscode folder.

- Within that folder, create a settings.json file and add the following:

{
    "liveServer.settings.https": {
    "enable": true,
    "cert": "<location of localhost.pem>",
    "key": "<location of localhost.key>",
    "passphrase": "<password when creating the localhost.key>"
    }
}

- You may get a warning saying the certificate is not secure. Click advanced and go continue to the site.

> Assuming everything has been set up correctly, you should now see the Shaka Player working locally with https

Reviewer: ```suggestion
Assuming everything has been set up correctly, you should now see the Shaka Player working locally with https.
```

Author: Actioned.

> By default, Content Name is set to “vualto-drm-encrypted-video”

Reviewer: ```suggestion
By default, Content Name is set to “vualto-drm-encrypted-video”.
```

Author: Actioned.
